pkg/crawl/crawler.go

```diff
@@ -114,3 +114,9 @@ func (c Crawler) includeDir(path string) bool {
 	}
 	return true
 }
+
+func (s *Stats) Append(stats Stats) {
+	s.FilesScanned = s.FilesScanned + stats.FilesScanned
```

Comment:
nit, but I think that `+=` here would make it easier to read (also reduces possible typo error surface)